### PR TITLE
Improve javadocs for querydsl-maven-plugin

### PR DESCRIPTION
**Pull request by timowest**
related to https://github.com/querydsl/querydsl/issues/1190

### CODE REVIEW

File: querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractExporterMojo.java [hunk 1]
```diff
@@ -33,35 +33,45 @@
...
+ * {@code AbstractExporterMojo} calls the {@link GenericExporter} tool using the
  * classpath of the module the plugin is invoked in.
  *
  */
 public abstract class AbstractExporterMojo extends AbstractMojo {
 
     /**
+     * target folder for sources
+     *
      * @parameter
```
Comment (johnktims) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractExporterMojo.java]: Shouldn't this just be `@param`?

Comment (Shredder121) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractExporterMojo.java]: No, this is a maven configuration doclet.

File: querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java [hunk 2]
```diff
@@ -156,21 +160,29 @@
...
 
     /**
+     * switch for {@code toString} addition
+     *
      * @parameter default-value=false
      */
     private boolean beanAddToString;
 
     /**
+     * switch for full construction addition
```
Comment (Shredder121) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java]: `construction` :arrow_right:  `constructor`?

File: querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java [hunk 3]
```diff
@@ -197,61 +209,85 @@
...
 
     /**
+     * switch for generating scala sources
+     *
      * @parameter default-value=false
      */
     private boolean createScalaSources;
 
     /**
+     * switch for using schema as suffix in package generation
```
Comment (Shredder121) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java]: Maybe add an example here?

File: querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java [hunk 4]
```diff
@@ -197,61 +209,85 @@
...
+     *
      * @parameter default-value=false
      */
     private boolean schemaToPackage;
 
     /**
+     * switch to lowercase name
```
Comment (Shredder121) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java]: `Switch to output name` (I would clarify this) `as lowercase`

File: querydsl-maven-plugin/src/main/java/com/querydsl/maven/MetadataExportMojo.java [hunk 5]
```diff
@@ -13,8 +13,10 @@
  */
 package com.querydsl.maven;
 
+import com.querydsl.sql.codegen.MetaDataExporter;
+
 /**
- * MetadataExportMojo is a goal for MetaDataExporter usage and is bound to the generate-sources phased
+ * {@code MetadataExportMojo} is a goal for {@link MetaDataExporter} usage and is bound to the generate-sources phased
```
Comment (Shredder121) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/MetadataExportMojo.java]: I'd reword this, at the end is also a typo.

**Comment by timowest** [comment]
I made some improvements.
